ds000117/.bidsignore

```diff
@@ -1,238 +1,2 @@
-sub-13/ses-mri/anat/sub-13_ses-mri_run-2_echo-2_FLASH.nii.gz
-sub-13/ses-mri/anat/sub-13_ses-mri_run-2_echo-3_FLASH.nii.gz
-sub-13/ses-mri/anat/sub-13_ses-mri_run-2_echo-4_FLASH.nii.gz
-sub-13/ses-mri/anat/sub-13_ses-mri_run-2_echo-5_FLASH.nii.gz
-sub-13/ses-mri/anat/sub-13_ses-mri_run-2_echo-6_FLASH.nii.gz
-sub-13/ses-mri/anat/sub-13_ses-mri_run-2_echo-7_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-1_echo-1_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-1_echo-2_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-1_echo-3_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-1_echo-4_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-1_echo-5_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-1_echo-6_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-1_echo-7_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-2_echo-1_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-2_echo-2_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-2_echo-3_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-2_echo-4_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-2_echo-5_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-2_echo-6_FLASH.nii.gz
-sub-14/ses-mri/anat/sub-14_ses-mri_run-2_echo-7_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-1_echo-1_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-1_echo-2_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-1_echo-3_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-1_echo-4_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-1_echo-5_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-1_echo-6_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-1_echo-7_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-2_echo-1_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-2_echo-2_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-2_echo-3_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-2_echo-4_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-2_echo-5_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-2_echo-6_FLASH.nii.gz
-sub-15/ses-mri/anat/sub-15_ses-mri_run-2_echo-7_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-1_echo-1_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-1_echo-2_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-1_echo-3_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-1_echo-4_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-1_echo-5_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-1_echo-6_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-1_echo-7_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-2_echo-1_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-2_echo-2_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-2_echo-3_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-2_echo-4_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-2_echo-5_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-2_echo-6_FLASH.nii.gz
-sub-16/ses-mri/anat/sub-16_ses-mri_run-2_echo-7_FLASH.nii.gz
+run-*_echo-*_FLASH.json
```

Comment:
@chrisfilo I am not 100% sure but it seems to me that number ranges like [0-9] is not working here. So, I used the more permissive `*` for now instead of `[1-16]` or so

Reply:
sure!